### ISSUE TITLE
arwm.m still relies on obsolete matlabpool command

### ISSUE THREAD

**Issue by ripatti** [IRIS-Solutions-Team/IRIS-Toolbox#228]
arwm offers an option to use prefetching in MH. The code still relies on matlabpool() command that was removed from R2013b onwards.

**Comment by jaromir-benes** [comment]
Antti - the prefetchin algorithm actually never really delivered tangible
benefits in terms of accelerating the calculations, so we at some point
stopped maintaining it...

**Comment by ripatti** [comment]
Indeed. This is my, limited, experience too. No need to maintain. Code can still be there for youngsters who do not believe :)

**Comment by jaromir-benes** [comment]
Haha, my thoughts exactly :)
